NguyenVanViet2004/FE-GlowUp
# Implement Notification UI

## Related issues
Resolved #69 

## Summary
- Implement the Notification  UI
- Implement the Change Password UI
- Update View Booking

## How to test
- Run the app and tap on your notification icon
- Go to Profile and select change password
- Go to booking and click on view voucher button (make sure your account has data or you can log in with account {acccount: 0327367649, password: 123456})

## Concern points

## Screenshots
| --- | --- | --- |
| --- | --- | --- |
|![image](https://github.com/user-attachments/assets/589ca880-d9d7-40b0-9631-9e1952f6c5ea)|![image](https://github.com/user-attachments/assets/6c24a12c-a731-45ca-a4a6-f53c67e3d385)|![image](https://github.com/user-attachments/assets/98cc0b2c-b7a1-4d52-ad16-9ff848bad09c)|
